joplin/base/models/service_page.py

```diff
@@ -19,7 +19,7 @@
 from countable_field import widgets
 
 
-WYSIWYG_SERVICE_STEP = ['ul', 'link', 'code', 'rich-text-button-link']
+WYSIWYG_SERVICE_STEP = ['ul', 'link', 'code', 'rich-text-button-link', 'document-link']
```

Comment:
the one change I did, the rest of the files changed are from merging the 2.7 branch